frontend/src/interfaces/releasePlans.ts

```diff
@@ -21,19 +19,7 @@ export interface IReleasePlanTemplate {
     archivedAt?: string;
 }
 
-export interface ISafeguard {
```

Comment:
Removed manual model and using orval types.

Reply:
nice